packages/sales-channel/src/models/sales-channel.ts

```diff
@@ -0,0 +1,50 @@
+import { generateEntityId } from "@medusajs/utils"
+import {
+  BeforeCreate,
+  Entity,
+  Index,
+  OptionalProps,
+  PrimaryKey,
+  Property,
+} from "@mikro-orm/core"
+
+@Entity()
+export default class SalesChannel {
+  [OptionalProps]?: "created_at" | "updated_at" | "deleted_at"
+
+  @PrimaryKey({ columnType: "text" })
+  id!: string
+
+  @Property({ columnType: "text" })
+  name!: string
+
+  @Property({ columnType: "text" })
+  description!: string
```

Comment:
suggestion: Make optional
```suggestion
  @Property({ columnType: "text", nullable: true })
  description?: string | null
```